llvm-runtimes/compiler-rt-sanitizers/compiler-rt-sanitizers-22.0.0_pre20250820.ebuild

```diff
@@ -117,6 +120,17 @@ src_configure() {
 		local -x CC=${CHOST}-clang
 		local -x CXX=${CHOST}-clang++
 		strip-unsupported-flags
+
+		# The full clang configuration might not be ready yet. Use the partial
+		# configuration files that are guaranteed to exist even during initial
+		# installations and upgrades.
+		local flags=(
+			--no-default-config
+			"--config=${ESYSROOT}/etc/clang/${LLVM_MAJOR}/gentoo-{rtlib,stdlib,linker}.cfg"
```

Comment:
You can't have the `{...}` expansion inside `""`.

Reply:
Good catch and sorry! Fixed now.